### PR TITLE
update release note for bug list

### PR DESCRIPTION
**Pull request by YuzuruUeda**

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -21,6 +21,7 @@ Enhancements
 
 Bug Fixes
 ~~~~~~~~~
+* 9937 segmentation fault when generating a machine image
```
Comment (emuus) [inline changelog.rst]: Capital S at start of bug description

File: changelog.rst [hunk 2]
```diff
@@ -19,6 +19,38 @@ Enhancements
 * Allow users to delete multiple migrations
 * 9057 Remove End-of-life Microsoft Azure Classic
 
+Bug Fixes
+~~~~~~~~~
+* 9762 Default chunk size for publishing to K5 is too low and leads to K5 error
+* 9411 When scaning a machine, UForge portal UI displays a big OS Logo in IE fix the logo size in the header + allow to debug CSS from remote computers
```
Comment (emuus) [inline changelog.rst]: scanning (2 n). ... displays a big OS logo. Fix the logo size in the header and allow ...

File: changelog.rst [hunk 3]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9771 Docker image format should not be proposed for Windows migrations
+* 9781 When selecting Azure platform, AWS or K5 for generation, an unknown servor error is displayed
```
Comment (emuus) [inline changelog.rst]: server (typo)

File: changelog.rst [hunk 4]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9424 When a migration is deleted, the error message of the migration tool is not relevant
+* 9641 Delete account with a certificate, after a publish, fails
+* 9639 Wrong URI for Scan installProfile
+* 9657 Unable to publish Docker images to Azure Containers
+* 9615 Generation dashboard is in error after a Migration Generation Stage has completed
+* 8917 After Blackbox Migration, RHEL5.3 were updated to RHEL5.5.
```
Comment (emuus) [inline changelog.rst]: RHEL 5.3 is updated

File: changelog.rst [hunk 5]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9598 provisioning on Azure fails due to No DVD device
```
Comment (emuus) [inline changelog.rst]: start with capital

File: changelog.rst [hunk 6]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9582 SLES generation for Azure format does not work (no platform tools injected)
+* 9258 Unable to do a Debian 8 blackbox migration to Microsoft Azure
+* 9074 Improve error handling in getCredAccountResources service
+* 9188 partitions in LVM logical groups/volumes appear in wrong order if a group's name is changed
```
Comment (emuus) [inline changelog.rst]: start with capital

File: changelog.rst [hunk 7]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9532 When importing an appliance, Firewall is set to Ask during installation
+* 9502 NIC configuration disappears on install profile in an imported appliance from a CentOS5.7 scan
+* 8682 if the root context is not modified in the deployment wizard, the page redirected to at the end is / which is forbidden
```
Comment (emuus) [inline changelog.rst]: start with capital. not sure I understand which page is redirect to ...

File: changelog.rst [hunk 8]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9081 "should contains" to be replaced with "should contain" in bootscript name in UI
+* 9500 Fix random should_delete_build_query_params test
+* 9498 ow2stack.org is down and an IT test use it Temporary remove the tests
```
Comment (emuus) [inline changelog.rst]: Not sure this is a public bug? If so no written clearly

File: changelog.rst [hunk 9]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9598 provisioning on Azure fails due to No DVD device
+* 9582 SLES generation for Azure format does not work (no platform tools injected)
+* 9258 Unable to do a Debian 8 blackbox migration to Microsoft Azure
+* 9074 Improve error handling in getCredAccountResources service
+* 9188 partitions in LVM logical groups/volumes appear in wrong order if a group's name is changed
+* 9532 When importing an appliance, Firewall is set to Ask during installation
+* 9502 NIC configuration disappears on install profile in an imported appliance from a CentOS5.7 scan
+* 8682 if the root context is not modified in the deployment wizard, the page redirected to at the end is / which is forbidden
+* 9081 "should contains" to be replaced with "should contain" in bootscript name in UI
+* 9500 Fix random should_delete_build_query_params test
```
Comment (emuus) [inline changelog.rst]: is this a public bug?

File: changelog.rst [hunk 10]
```diff
@@ -19,6 +19,38 @@ Enhancements
...
+* 9498 ow2stack.org is down and an IT test use it Temporary remove the tests
+* 6200 fix scan installProfile URI
```
Comment (emuus) [inline changelog.rst]: start with capital

File: changelog.rst (hunk 2 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fixed

File: changelog.rst (hunk 3 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fxed

File: changelog.rst (hunk 4 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fixed

File: changelog.rst (hunk 5 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fixed

File: changelog.rst (hunk 6 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fixed

File: changelog.rst (hunk 7 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fixed to start with capital.
At this moment, I am also not quite sure which page is redirect to...

File: changelog.rst (hunk 8 above)
Comment (YuzuruUeda) [reply changelog.rst]: Removed, as it is not public one.

File: changelog.rst (hunk 9 above)
Comment (YuzuruUeda) [reply changelog.rst]: Removed, as it is not public one.

File: changelog.rst (hunk 10 above)
Comment (YuzuruUeda) [reply changelog.rst]: Fixed

**Pull request merged**